src/option.rs

```diff
@@ -819,8 +850,9 @@ mod test {
 
         assert_parse!(r#"User dusty"#, SSHOption::User(String::from("dusty")));
         assert_parse!(r#"Port 22"#, SSHOption::Port(22));
-        // TODO: getservbyname
-        // assert_parse!(r#"Port ssh"#, SSHOption::Port(22));
+        if cfg!(feature = "with_libc") {
+            assert_parse!(r#"Port ssh"#, SSHOption::Port(22));
```

Comment:
Why does this pass, but the integration test that tries to get `ssh` to do the same thing doesn't?

Reply:
Oh because that code is relatively recent: https://github.com/openssh/openssh-portable/commit/2581333d564d8697837729b3d07d45738eaf5a54